ansible/services/nix-komodo/.env

```
@@ -0,0 +1,130 @@
+## Rate Komodo polls Resources for updates,
+## like outdated commit hash.
+## Options: 1-min, 5-min, 15-min, 30-min, 1-hr.
+## Default: 5-min
+KOMODO_RESOURCE_POLL_INTERVAL="5-min"
+
+## Used to auth against periphery. Alt: KOMODO_PASSKEY_FILE
+KOMODO_PASSKEY=${PASSKEY}
+## Used to auth incoming webhooks. Alt: KOMODO_WEBHOOK_SECRET_FILE
+KOMODO_WEBHOOK_SECRET=a_random_secret
+## Used to generate jwt. Alt: KOMODO_JWT_SECRET_FILE
+KOMODO_JWT_SECRET=a_random_jwt_secret
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The KOMODO_JWT_SECRET key should go before the KOMODO_PASSKEY key

(UnorderedKey)